net/net.h

```diff
@@ -1752,8 +1752,20 @@ M0_TL_DECLARE(m0_net_tm, M0_INTERNAL, struct m0_net_buffer);
  *     ipv4addr = 1*3DIGIT "." 1*3DIGIT "." 1*3DIGIT "." 1*3DIGIT ; 0..255
  */
 M0_INTERNAL bool m0_net_endpoint_is_valid(const char *endpoint);
-#endif
 
+#endif
+/** Set default network transport as lnet. */
+M0_INTERNAL void m0_net_xprt_default_set(const struct m0_net_xprt *xprt);
+/** Register network transport. */
```

Comment:
It's better to leave a blank line between functions.
But this is my personal preference.